src/services/infra/InfraService.ts

```diff
@@ -16,9 +20,10 @@ import {
 import ReposService from "@services/identity/ReposService"
 import SitesService from "@services/identity/SitesService"
 
-import QueueService, { MessageBody } from "../identity/QueueService"
+import QueueService from "../identity/QueueService"
 
 const SITE_LAUNCH_UPDATE_INTERVAL = 30000
+export const REDIRECTION_SERVER_IP = "18.136.36.203"
```

Comment:
wait what - is this guaranteed?

Reply:
yes, but have removed this as since its not being used!